service-commands/src/commands/service-commands.json

```diff
@@ -96,6 +96,12 @@
     "down": [
       "cd creator-node; . compose/env/shellEnv#.sh; docker-compose -f compose/docker-compose.yml down --remove-orphans"
     ],
+    "restart": [
```

Comment:
restarting a service with down & up is broken bc it doesn't update-cnode-config. this is a convenience command to get around that